Cohort2/team4/client/src/pages/events/index.tsx

```diff
@@ -1,13 +1,28 @@
-import EventList from '../../components/EventList';
+import EventList from "../../components/EventList";
 
 export default function EventDashboard() {
-	const events = ['event1'];
-	return (
-		<main className="flex min-h-screen flex-col items-center justify-between p-24">
-			<div>
-				<div>TODO -- Replace with event dashboard</div>
-				<EventList events={events}/>
-			</div>
-		</main>
-	);
+  const user = "Sally";
+  const events = ["event1"];
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+      <div>
+        <header className="flex items-center gap-[70vw]">
+          Convi
+          <button className="bg-[#a6c1ee] text-white px-5 py-2 rounded-full hover:bg-[#87acec]">
+            Sign in
```

Comment:
Nit: Technically at this point we'd be signed in, as its required to see the Event page. So this should probably be "Sign out".

Reply:
That's true. I will fix it.